lib/iris/analysis/__init__.py

```diff
@@ -348,14 +349,16 @@ def coord_comparison(*cubes):
     return result
 
 
-class Aggregator(object):
+class _Aggregator(object):
     """
-    The :class:`Aggregator` class provides common aggregation functionality.
+    The :class:`_Aggregator` abstract base class provides common
+    aggregation functionality.
 
     """
+    __metaclass__ = ABCMeta
```

Comment:
@pelson - I don't share your blanket aversion. ABC can be useful when writing subclasses.